src/views/Plant.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="plant-page">
+    <div class="header">
+      <BaseButton
```

Comment:
probabilmente ha senso fare un `Componente` chiamato tipo `TabBar` che contiene più pulsanti in modo modulate ... tipo gli puoi dire quanti ne deve contenere ( in questo caso due ) e con quello gestire il cambiamento di stato. Questo anziché mettere i due `BaseButton` all'interno di questa `View`. Un buon motivo sarebbe quello ce quella `TabBar` sarebbe ricusabile altrove e in questa pagina ci sarebbe più ordine, anche con la visione di aggiungere magari più pulsanti.